lib/miq_report_formats.rb

```diff
@@ -0,0 +1,44 @@
+class MiqReportFormats
```

Comment:
I don't even see MiqReport as a model to be honest :smile:  As this is a subcomponent of MiqReport, I see it closer as a private implementation for formats, and as such I feel like it belongs as a nested class under the model.